pkg/dsdk/remote_provider.go

```diff
@@ -229,3 +229,27 @@ func (e *RemoteProvider) Reload(ro *RemoteProviderReloadRequest) (*RemoteProvide
 	RegisterRemoteProviderEndpoints(resp)
 	return resp, nil, nil
 }
+
+type RemoteProviderOperationsUpdateRequest struct {
+	Ctxt        context.Context `json:"-"`
+	OperationId string          `json:"omit"`
+	Action      string          `json:"action"` //available options are 'clear' and 'abort'
+}
+
+func (e *RemoteProvider) PerformOperation(ao *RemoteProviderOperationsUpdateRequest) (*RemoteProvider, *ApiErrorResponse, error) {
+
+	gro := &greq.RequestOptions{JSON: ao}
```

Comment:
@tjcelaya: is this the right way to handle this? the omit should remove the OperationId from the JSON serialization, which is what we want

Reply:
You need to switch to a hypen like the context field, the syntax you're using indicates that the field name would "omit"